rpcs3/rpcs3qt/debugger_frame.cpp

```diff
@@ -358,7 +365,7 @@ void debugger_frame::keyPressEvent(QKeyEvent* event)
 		case Qt::Key_M:
 		{
 			// Memory viewer
-			idm::make<memory_viewer_handle>(this, pc, cpu);
+			if (m_cpu) idm::make<memory_viewer_handle>(this, pc, m_cpu);
```

Comment:
this changes the cpu object to m_cpu. was this intended or is this copy pasta from renaming cpu to m_cpu?